driver-adapters/neon-http-vercel-nextjs-edgemw/next.config.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * @type {import('next').NextConfig}
+ */
+const nextConfig = {
+  webpack(config) {
+    config.experiments ??= {}
+
+    config.experiments.asyncWebAssembly = true
```

Comment:
Always needed for our users?

Reply:
Yes, unfortunately. Without it, you get an error that tells you to add it.